django/core/mail/backends/smtp.py

```diff
@@ -48,23 +75,36 @@ def open(self):
             # Nothing to do if the connection is already open.
             return False
 
+        # Create a SSLContext object with default settings and CA parameters.
+        context = ssl.create_default_context(
+            cafile=self.ssl_cafile,
+            capath=self.ssl_capath,
+            cadata=self.ssl_cadata
```

Comment:
Include a trailing comma in cases like this so that if more items are added later, this line doesn't have to be modified again.